deepspeed/autotuning/autotuner.py

```diff
@@ -23,6 +23,9 @@
 
 
 class Autotuner:
+    """The DeepSpeed Autotuner automatically dicover the optimal DeepSpeed configuration that delievers good training speed. The Autotuner uses model information, system information, and heuristics to efficiently tune system knobs that affect compute and memory efficiencies, such as ZeRO optimization stages, micro-batch sizes, and many other ZeRO optimization configurations. It not only saves users' time but also can discover configurations better than hand-tuned method.
+    Autotuing with DeepSpeed requires no code change from DeepSpeed users. Please refer to the README for usage detils.
```

Comment:
It not only reduces the time and resources user spend on tuning, but can

Reply:
Can you elaborate more on this?

Reply:
Sure. The main suggestion is to point out that autotuner also saves user resources and not just time. 

And I suggested that as a rewrite of "It not only saves users' time but also can". But  you ignore the rewrite suggestion, but please include the resource saving benefit.

Reply:
I see. Just made the change. Thanks.